applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -29,10 +29,10 @@ LOG_MODULE_REGISTER(cis_headset, CONFIG_LOG_BLE_LEVEL);
 			BT_GAP_ADV_FAST_INT_MAX_1, NULL)
 
 /* Advertising data for peer connection */
-
 static const struct bt_data ad_peer[] = {
 	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
-	BT_DATA(BT_DATA_NAME_COMPLETE, DEVICE_NAME_PEER, DEVICE_NAME_PEER_LEN),
+	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_ASCS_VAL)),
+	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_PACS_VAL))
```

Comment:
I see that this is a little different from the unicast_audio_server Zephyr sample.
How did you choose these parameters?

Reply:
I take the reference from BAP v1.0.1, 8.1.1.1 "Connection procedure to non-bonded devices"
![image](https://user-images.githubusercontent.com/23454299/194465907-cef216d0-ef08-4558-866b-e41d2001b29c.png)
Which expose the ASCS and PACS, coincided to what we seen in the UPF.